test/csv/csv-filter/test_csv_filter.rb

```diff
@@ -0,0 +1,102 @@
+# frozen_string_literal: false
+
+require_relative '../helper'
+
+require 'csv'
```

Comment:
Could you remove this because `../helper` has this?

Reply:
Removed.